main.go

```diff
@@ -1,77 +1,165 @@
 package main
 
 import (
-	"flag"
+	"fmt"
 	"log"
-	"sync"
+	"time"
+
+	"github.com/Regner/albionmarket-client/asserts"
+	"github.com/Regner/albionmarket-client/config"
+	"github.com/Regner/albionmarket-client/operations"
+	"github.com/Regner/albionmarket-client/utils"
```

Comment:
When you make this change you will need to make sure to change your directory on disk.